### PR TITLE
Update NetworkLevelPrivacy.md

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

File: docs/why-wasabi/NetworkLevelPrivacy.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ It is also possible to connect Wasabi to a remote full node on another computer
 Then, Wasabi pulls the verified blocks and queries the mempool from the full node.
 
 :::tip Wasabi ships with Bitcoin Knots!
-Since v1.1.11 Wasabi has [partial Bitcoin Knots integration](/using-wasabi/BitcoinFullNode.md).
+Wasabi has [partial Bitcoin Knots integration](/using-wasabi/BitcoinFullNode.md).
```
Comment (yahiheb) [inline docs/why-wasabi/NetworkLevelPrivacy.md:49]:
```suggestion
Wasabi has a [partial Bitcoin Knots integration](/using-wasabi/BitcoinFullNode.md).
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]

File: docs/why-wasabi/NetworkLevelPrivacy.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ It is also possible to connect Wasabi to a remote full node on another computer
 Then, Wasabi pulls the verified blocks and queries the mempool from the full node.
 
 :::tip Wasabi ships with Bitcoin Knots!
-Since v1.1.11 Wasabi has [partial Bitcoin Knots integration](/using-wasabi/BitcoinFullNode.md).
+Wasabi has a [partial Bitcoin Knots integration](/using-wasabi/BitcoinFullNode.md).
 This means that it is possible (but not mandatory) to start Bitcoin Knots during the launch of Wasabi.
 Without having to install or configure anything.
```
Comment (BTCparadigm) [inline docs/why-wasabi/NetworkLevelPrivacy.md:51]:
```suggestion
This means that it is possible (but not mandatory) to start Bitcoin Knots during the launch of Wasabi, without having to install or configure anything manually.
```

File: docs/why-wasabi/NetworkLevelPrivacy.md [hunk 3]
```diff
@@ -132,7 +132,7 @@ Wasabi broadcast transactions to only one peer over Tor, and immediately after t
 ### Wasabi Wallet + full node
 
 :::tip
-Since version [1.1.11](https://github.com/zkSNACKs/WalletWasabi/releases/tag/v1.1.11), Wasabi has partial Bitcoin Knots integration.
+Wasabi has a partial Bitcoin Knots integration.
 This means that it is possible (but not mandatory) to start Bitcoin Knots during the launch of Wasabi, without having to install or configure anything.
```
Comment (BTCparadigm) [inline docs/why-wasabi/NetworkLevelPrivacy.md:136]:
```suggestion
This means that it is possible (but not mandatory) to start Bitcoin Knots during the launch of Wasabi, without having to install or configure anything manually.
```

**Review by BTCparadigm — COMMENTED** [review]
Here's my nitpicks

**Review by yahiheb — APPROVED** [review]

**Pull request merged**